application/common/config/main.php

```diff
@@ -101,6 +101,7 @@
                     'logVars' => [],
                     'categories' => ['application'],
                     'prefix' => $logPrefix,
+                    'exportInterval' => 1,
```

Comment:
"How many messages should be accumulated before they are exported. Defaults to 1000."
https://www.yiiframework.com/doc/api/2.0/yii-log-target#$exportInterval-detail